config/routes.rb

```diff
@@ -1,7 +1,10 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
-  devise_for :employees
+  devise_for :admins, path: 'admins'
+  devise_for :employees, path: 'employees'
   resources :kudos
+  get '/employee' => "kudos#index", :as => :employee_root
```

Comment:
po co dodałeś route dla employaa do metody `kudos#index`, nie ma tego w wymaganiach zadania, a do metody `kudos#index` prowadzi już `resources :kudos`

Reply:
ponieważ to generowało ten błąd w rspecu który Ci wysłałem, ten routing error

Reply:
problem polegał na tym że w teście otwierasz adres `/employee`, a takiego nie masz, ale czy na pewno w teście chcesz wchodzić w employee w tym teście, w którym edytujesz kudo?

Zapoznaj się z REST URL Naming convention i powiedz czego powininieneś się spodziewać pod ścieżką `/employee`

Reply:
pamiętaj że pracodawca będzie od ciebie wymagał robienia dokładnie tego co podał w acceptance criteria, takie żeczy jak dodatkowe ścieżki itp należy uzgodnić i potrafić dokładnie uargumentować dlaczego tak zrobiliśmy